### DISCUSSION TITLE
Icon request: fork / split

### DISCUSSION THREAD

**Discussion by dorthrithil** [vmware/clarity#5881] [Ideas]
An icon for forking / splitting something would be useful.

**Example:**

![Screenshot 2021-04-20 at 15 12 17](https://user-images.githubusercontent.com/3655915/115401609-e1581f00-a1ea-11eb-85f0-ea57cda3ad78.png)

**Usecase:**

 - Version control systems
 - Many other cases where some existing entity is cloned with the purpose to change it (using the clone icon will be confusing as no exact copies are wanted)

**Comment by mathisscott** [comment]
Thanks, @dorthrithil ! We will look into this for our next batch of icons.

https://github.com/vmware/clarity/issues/5882
